### PR TITLE
Require php 7.4 and update dependencies

### PR DESCRIPTION
**Pull request by MortalFlesh**

### CODE REVIEW

**Comment by MortalFlesh** [comment]
First we need to merge https://github.com/lmc-eu/api-filter/pull/71 and then rebase it, so the github actions will run in this PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 <!-- There should always be "Unreleased" section at the beginning. -->
 
 ## Unreleased
+- [**BC**] Require php 7.4 and update dependencies
```
Comment (OndraM) [inline CHANGELOG.md]: Requiring new PHP version is not a BC break, as you don't (I guess) change the library public API.

File: ecs.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        ['SlevomatCodingStandard\Sniffs\Exceptions\ReferenceThrowableOnlySniff.ReferencedGeneralException' => ['tests/Exception/*.php']]);
+
+    $containerConfigurator->import(__DIR__ . '/vendor/lmc/coding-standard/ecs.php');
+
+    $services = $containerConfigurator->services();
+
+    $services->set(PhpUnitTestAnnotationFixer::class)
+        ->call('configure', [['style' => 'annotation']]);
+
+    $services->set('Symplify\CodingStandard\Sniffs\Naming\ClassNameSuffixByParentSniff')
```
Comment (OndraM) [inline ecs.php]: `ClassNameSuffixByParentSniff::class`?

File: ecs.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+use PhpCsFixer\Fixer\Phpdoc\NoSuperfluousPhpdocTagsFixer;
+use PhpCsFixer\Fixer\PhpUnit\PhpUnitTestAnnotationFixer;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $parameters = $containerConfigurator->parameters();
+
+    $parameters->set('skip',
+        ['SlevomatCodingStandard\Sniffs\Exceptions\ReferenceThrowableOnlySniff.ReferencedGeneralException' => ['tests/Exception/*.php']]);
```
Comment (OndraM) [inline ecs.php]: Weird codestyle...

File: ecs.php [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+declare(strict_types=1);
+
+use PhpCsFixer\Fixer\Phpdoc\NoSuperfluousPhpdocTagsFixer;
+use PhpCsFixer\Fixer\PhpUnit\PhpUnitTestAnnotationFixer;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $parameters = $containerConfigurator->parameters();
+
+    $parameters->set('skip',
```
Comment (OndraM) [inline ecs.php]:
```suggestion
    $parameters->set(Option::SKIP,
```

(and `use Symplify\EasyCodingStandard\ValueObject\Option`)

**Review by legendik — APPROVED** [review]

**Pull request merged**